everviz/plugins/_crossplot/crossplot.py

```diff
@@ -0,0 +1,134 @@
+import os
+import pkg_resources
+import dash_html_components as html
+import flask
```

Comment:
Should we add an entry for flask in the list of install requirements for everviz?